neurots/schemas/parameters.json

```diff
@@ -117,6 +117,38 @@
                                             },
                                             "additionalProperties": false,
                                             "required": ["primary_orientation"]
+                                         },
+                                          {
+                                            "type": "object",
+                                            "properties": {
+                                                "direction": {
```

Comment:
in the code I have direction=[0, 1] for example, would this work?

Reply:
You would have to tranform the dict into list but I think it would be more explicit for the user to enter an object with named values instead of a list of 2 values with implicit meaning (mean and std respectively).

Reply:
yes, you're right, I can try that

Reply:
done!